srcopsmetrics/entities/README.rst

```diff
@@ -0,0 +1,134 @@
+====================================
+Meta-information Indicators Entities
+====================================
+
+Entity Criteria
+===============
+
+See template.py for necessary functionality that has to be provided in each Entity implementation.
+
+
+Meta-information Indicators Metrics
+===================================
+For every repository we want to gather following metrics:
+
+
+Time to Close an Issue (TTCI)
+-----------------------------
+For every closed Issue in repository.
+
+
+Median Time to Close an Issue (MTTCI)
+------------------------------------------
+Each Issue TTCI can be compared to the metrics of Issues closed before.
+Therefore, median TTCI for every Issue in repository.
+
+
+Time to Close an Issue Score (TTCI_SCORE)
```

Comment:
also here should be `TTCIS` if you are using acronyms, same below

Reply:
renamed